src/main/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/ValidationModelService.java

```diff
@@ -93,7 +93,8 @@ public void validateRequest(AssignmentRequest assignmentRequest) {
     public List<Assignment> getCurrentRoleAssignmentsForActors(Set<String> actorIds) {
         var queryRequest = QueryRequest.builder()
             .actorId(actorIds)
-            .roleType("ORGANISATION")
+            .roleType(List.of("ORGANISATION", "CASE"))
+            .grantType(List.of("STANDARD", "NONE"))
```

Comment:
Need to include BASIC as well in the grantType list. otherwise some scenario will fail. please cover this in the jUnit as well.